File: frontend/kotlin/working_notes/NoteApplication/app/src/main/java/com/example/noteapplication/features/login/domain/models/LoginResponse.kt
package com.example.noteapplication.features.login.domain.models

import com.google.gson.annotations.SerializedName

data class LoginResponse (
    @SerializedName("access_token")
    var accessToken: String,

    @SerializedName("expires_in")
    var expiresIn: String ,

    @SerializedName("token_type")
    var tokenType: String,

    @SerializedName("scope")
    var scope: String
)

File: frontend/flutter/notes_application/lib/features/login/domain/entities/login_response.dart
import 'package:equatable/equatable.dart';
import 'package:flutter/cupertino.dart';

class LoginResponse extends Equatable {
  final String accessToken;
  final int expiresIn;
  final String tokenType;
  final String scope;

  LoginResponse({
    @required this.accessToken,
    @required this.expiresIn,
    @required this.tokenType,
    @required this.scope,
  }) : super([accessToken, scope]);
}